afids_utils/tests/strategies.py

```diff
@@ -5,27 +5,29 @@
 from hypothesis.extra.numpy import arrays
 from numpy.typing import NDArray
 
-from afids_utils.afids import AfidPosition
+from afids_utils.afids import AfidPosition, AfidSet
 
 
 @st.composite
-def afid_coords(
+def afid_set(
```

Comment:
By convention, `SearchStrategy`s should have a plural name, so this should be called `afid_sets`